packages/klevu-core/src/utils/objectToQueryParamaters.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * transform object to query parameters
+ *
+ * @param params
+ */
+export function objectToQueryParameters(params: object) {
```

Comment:
Now that this has been moved to new file it should be used also in original place where it was copied from. So that we do not have duplicate implementation.